zulipterminal/helper.py

```diff
@@ -47,6 +47,9 @@
 )
 
 
+RESOLVED_TOPIC_PREFIX = "✔ "
+
+
```

Comment:
Adding new lines just increases the no. of changes. So, try minimizing your code changes by sticking to only code additions and omissions, not the spacing.

Reply:
Apologies, I should have said that this was already defined in api_types.py - it has to be consistent with the API for resolving to be recognized.